README.md

```diff
@@ -0,0 +1,117 @@
+(does __NOT__ guarantee correctness, just completion of the transformation code), failed, and
+files that were skipped. The skipping reason is logged, for example the test does not src `js-test.js`.
+The tool also lists whether transformed files succeeded or failed verification.
+With this summary output, you can manually check files failed transformations/verifications to see if the
+problem is easily fixed by a human, report bugs with this tool if the transformation should have worked, or just 
+`git checkout` those files and commit the succesful transformations to Gerrit.
+
+The tool currently only transforms .html tests. If a test src's a helper .js script, neither file will be 
+transformed and the test will be marked as skipped. If a test src's a helper .js script __and__ `js-test.js`,
+the test file will be transformed, marked as completed, but verification will fail.
+
+**Example**
+`$node ~/web-test-transfomer/index.js --dir=fast/dom/SelectorAPI --target_build=Debug`
+
+### Dry Run
+If you would like to do a dry run to compare the files, create a directory and pass the path
+to the `--output_dir` flag.
+
+
+### CLI Flags
+`$node index.js --help`
+
+```
+Options:
+  --file: Path to test file to transform
+    (default: null)
+  --dir: Path to dir of test files to transform
+    (default: null)
+  --output_dir: Path to dir where output files should be written. If 
+    null, will overwrite input files. 
+    (default: null)
+  --target_build: Target build used in -t parameter for run_web_tests.py
+    (default: "Default")
+  --[no]quiet: Disable logging
+    (default: false)
+  --[no]verify: Runs web test after transforming.
+    (default: true)
+```
+
+#### Required Flags
+You must specify **exactly one** of `--file` or `--dir`.
+
+## Verification
+
+The tool attempts to provide automated verification of the transformation unless `--noverify` or `--verify=false` is passed. This is done by running the [Chromium web tests](https://chromium.googlesource.com/chromium/src/+/master/docs/testing/web_tests.md) on the transformed tests and verifying the test still produces the same results.
+
+Some `run_web_tests.py` output will be shown when running verification, and you will see failures prefixed 
+by `verify:ERROR`.
+
+
+
+### All PASS Requirement
+Since comparing failures within a test from the different frameworks is hard, the tool __only__ verifies tests whose baseline file ({test name}-expected.txt) is all PASS. 
+The tool will still attempt to transform all files, but will only verify a subset. 
+
+If a test does not contain all PASS, verification is immediately reported as failed without running the web 
+tests. You can manually review the transformed file, run the web tests, and commit the transformation and the
+new baseline if they seem correct.
+
+Once a baseline file is deemed all PASS, it is deleted. This is because the `testharness.js` framework does
```

Comment:
we only delete the all-pass baseline if verification succeeds, right?

Reply:
Unfortunately no. The way I understood it: If we keep the baseline from js-test.js and then run the testharness.js test, it doesn't understand the expected file's contents since they use a different format. And testharness does not generate a baseline if the test succeeds.

Reply:
Deleting isn't a big deal since if something goes wrong you can always `git restore`